internal/v1/api.yaml

```diff
@@ -98,14 +115,14 @@ paths:
             default: 100
             minimum: 1
             maximum: 100
-          description: max amount of composes, default 100
+          description: "Max amount of composes, default 100."
```

Comment:
Does the description really need to also include the default? I see that the ReDoc service also shows the default so it's not needed there -- but can other API viewers be expected to do the same thing?